setup.py

```diff
@@ -1,16 +1,23 @@
 from setuptools import setup
-from eralchemy import __version__
 try:
     with open('readme.rst') as f:
         long_description = f.read()
 except IOError:
     with open('readme.md') as f:
         long_description = f.read()
 
+
+def read_version():
```

Comment:
In my view this is less idiomatic to Python, but it definitely looks like it will work, so I say if you like it better then go for it!